packages/connect-wallet-modal/src/components/WalletsModal/WalletsModal.tsx

```diff
@@ -133,12 +137,17 @@ export function WalletsModal(props: WalletsModalProps) {
         >
           <Terms {...termsProps} />
           <WalletsButtonsContainer $buttonsFullWidth={buttonsFullWidth}>
-            {props.children(buttonsCommonProps)}
+            {getWalletsButtons({
+              commonProps: buttonsCommonProps,
+              buttonComponentsByConnectorId,
+              hiddenWallets,
+              walletDataList,
+            })}
           </WalletsButtonsContainer>
         </Modal>
       );
     }
   }
-  // eslint-disable-next-line react/jsx-no-useless-fragment
+
   return <></>;
```

Comment:
You can return null for empty render. It's best practice.

Reply:
fixed